src/main/resources/SCHEMA.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE Users(
+                      id INT NOT NULL,
+                      username VARCHAR(64) NOT NULL,
+                      email VARCHAR(64) NOT NULL,
+                      password VARCHAR(64) NOT NULL,
+                      role VARCHAR(64) NOT NULL,
+                      created_at TIMESTAMP NOT NULL
+);
+CREATE TABLE Roles(
+    id INT NOT NULL,
```

Comment:
Please, indent the second table definition with the same indentation as the first one.